docs/architecture/adr-004-ics29-lock-fee-module.md

```diff
@@ -0,0 +1,55 @@
+# ADR 004: Lock fee module upon escrow out of balance
+
+## Changelog
+* 03/03/2022: initial draft
+
+## Status
+
+Accepted
+
+## Context
+
+The fee module maintains an escrow account for each of the fees escrowed to incentivize packet relays. 
```

Comment:
Aren't we just using a single account for all fees? This reads like each fee is getting its own separate account

Reply:
correct. Applied your suggestion